app/models/enhanced_loan_data_export.rb

```diff
@@ -112,12 +113,13 @@ def loan_data_as_hash(loan)
   end
 
   def response_hash(loan)
-    return {} if loan.criteria.blank?
     result = {}
-    response_set = loan.criteria
```

Comment:
lines 115 and 117 were the culprit - they trigger recursive building of the question tree from the database, which was happening twice per loan. we don't need question data here (on purpose) - we rely on the question ids in the raw response json. the new logic in lines 117, 118, and 119 let us access that response json w/out triggering the question-related db queries.